drv/Function.drv

```diff
@@ -104,9 +159,13 @@ public interface FUNCTION KEY_VALUE_GENERIC extends Function<KEY_GENERIC_CLASS,
 
 	/** {@inheritDoc}
 	 * @deprecated Please use the corresponding type-specific method instead. */
+
 	@Deprecated
 	@Override
-	boolean containsKey(Object key);
+	default boolean containsKey(Object key) {
```

Comment:
Yes, for primitives its clear. I just meant that in the case of, say, `Object2IntMap` would a `put(null, 1)` be something you want allow to happen? Then the `if (key == null) return false` check should actually only be happening for primitive keys and not all. In the abstract implementation, this was not the case (i.e. `null` keys were not considered by it). I am just not sure if this is intended. If I didn't miss it this is not explicitly documented.